src/Container/HStringContains.php

```diff
@@ -0,0 +1,48 @@
+<?php
```

Comment:
I don't know why git thinks this is created here and deleted later when this was just a simple rename.

There are also a few changes in the `contains` method that simplifies its behavior.